bundles/org.openhab.binding.freeathomesystem/src/main/resources/OH-INF/thing/thing-type.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="freeathomesystem"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<thing-type id="device" listed="false">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="gateway"/>
+		</supported-bridge-type-refs>
+		<label>free@home Device</label>
+		<description>This represents the generic free@home device</description>
+		<properties>
+			<property name="vendor">Busch&amp;Jäger - ABB</property>
+			<property name="modelId">Generic free@home Device</property>
+		</properties>
+		<representation-property>deviceId</representation-property>
+		<config-description>
+			<parameter name="deviceId" type="text" required="true">
+				<label>@text/thing-type.config.freeathomesystem.device.id</label>
+				<description>@text/thing-type.config.freeathomesystem.device.id.description</description>
```

Comment:
Please provide the texts directly here like in `bridge-type.xml` and let `mvn i18n:generate-default-translations` regenerate the I18N properties file. Both is possible, but should not be mixed.

Reply:
**Fixed**